.agents/skills/overdrive/SKILL.md

```
@@ -0,0 +1,142 @@
+---
+name: overdrive
+description: Pushes interfaces past conventional limits with technically ambitious implementations — shaders, spring physics, scroll-driven reveals, 60fps animations. Use when the user wants to wow, impress, go all-out, or make something that feels extraordinary.
+version: 2.1.1
+user-invocable: true
+argument-hint: "[target]"
+---
+
+Start your response with:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)